src/PostRector/Rector/NameImportingPostRector.php

```diff
@@ -42,6 +43,10 @@ public function enterNode(Node $node): Node|null
             return null;
         }
 
+        // notify this rule changing code
+        $rectorWithLineChange = new RectorWithLineChange(self::class, $node->getStartLine());
+        $this->file->addRectorClassWithLine($rectorWithLineChange);
```

Comment:
make importName() call to variable, if it is instance of Name, add to list, if it return null, just return as is.